tests/jballerina-unit-test/src/test/resources/test-src/imports/predeclared-imports/src/bar/predeclared-modules.bal

```diff
@@ -0,0 +1,81 @@
+// Copyright (c) 2020 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+//
+// WSO2 Inc. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+function testSumFunctionInDecimal(decimal p1, decimal p2) returns decimal {
+    return 'decimal:sum(p1, p2);
+}
+
+function testOneArgMaxFunctionInDecimal(decimal arg) returns decimal {
+    return 'decimal:max(arg);
+}
+
+function testSumFunctionInFloat() returns float {
+    return 'float:sum(12.34, 23.45, 34.56);
+}
+
+function testFloatConsts() returns float {
+    return 'float:NaN;
+}
+
+function testSumFunctionInInt() returns int {
+    return 'int:sum(10, 25, 35, 40);
+}
+
+function testMaxFunctionInInt(int n, int... ns) returns int {
+    return 'int:max(n, ...ns);
+}
+
+string str = "Hello Ballerina!";
+
+function testSubString() returns string {
+    return 'string:substring(str,6);
+}
+
+function testStartsWithFunctionInString() returns boolean {
+    return 'string:startsWith(str, "Hello");
+}
+
+function testPredeclaredModules() {
```

Comment:
We seem to have added tests for string, int, float, and decimal. Shall we add a simple test for the rest of the predeclared imports too?

Reply:
There are no use scenarios with other predeclared imports yet.

Reply:
We can still add tests for overriding, right?

Shall we also create an issue to add predeclared tests once the langlib content for these langlibs are introduced in the implementation?

Reply:
https://github.com/ballerina-platform/ballerina-lang/issues/25757

Reply:
@chiranSachintha, let's add few simple tests for overriding as discussed offline.